manifests/init.pp

```diff
@@ -1,14 +1,18 @@
 # == Class: lvm
```

Comment:
the old comment can also be deleted to please puppet-strings
```suggestion

```

Reply:
Updated